crates/collab_ui/src/chat_panel.rs

```diff
@@ -1096,6 +1096,10 @@ impl FocusableView for ChatPanel {
 }
 
 impl Panel for ChatPanel {
+    fn should_show_button(&self, cx: &gpui::WindowContext) -> bool {
```

Comment:
Currently we return `None` in icon if we need to hide a panel button.

Not sure if we need to add a new method for it. Perhaps it conveys the intent better.

Either way, we should be consistent for the various panels.

Reply:
I'll switch this to the existing convention, thanks for flagging